fera_full.py

```
@@ -1,3 +1,189 @@
-oid sha256:5ab83fbeaf0e929e2f0e95874fc498c5632859cc4ec44fdec4e826156be06ab2
-size 9622
+
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jun 23 17:07:07 2023
+
+@author: labinfo
+"""
+#codereview
+import multiprocessing as mp
+import sys, getopt, global_settings, indexador_fera, setproctitle, fera, utilities_general, global_settings, classes_general
```

Ruff

Redefinition of unused `global_settings` from line 10

Remove definition: `global_settings`

(F811)